source/index.md

```diff
@@ -1015,6 +1015,12 @@ https://gateway.clearhaus.com/account
 
 ## Changes
 
+### Add credits resource
+
+Starting 2018-xx-xxTxx:xx:xx+00:00 the `/credits` resource has been added,
+enabling creation of credits without card tokenization. This is an essential
+addition, as card tokenization is deprecated.
```

Comment:
Uhm, I think the more important note is that `/cards/:id/credits` will also be deprecated — and what the deadline for this functionality is.